src/edit_mode/vi/parser.rs

```diff
@@ -101,6 +101,7 @@ impl ParsedViSequence {
             (Some(Command::EnterViInsert), ParseResult::Incomplete)
             | (Some(Command::EnterViAppend), ParseResult::Incomplete)
             | (Some(Command::ChangeToLineEnd), ParseResult::Incomplete)
+            | (Some(Command::ChangeInside(_)), ParseResult::Incomplete)
```

Comment:
Not sure how much of a deal breaker it is to land, but it will toggle into insert mode even if the delimiter is not found.

Reply:
I added another check to make sure it was comparing against a valid change inside character before switching to insert mode.